kroxylicious-runtime/src/main/java/io/kroxylicious/proxy/internal/FilterHandler.java

```diff
@@ -71,17 +72,24 @@ public class FilterHandler extends ChannelDuplexHandler {
     private final VirtualClusterModel virtualClusterModel;
     private final Channel inboundChannel;
     private final FilterAndInvoker filterAndInvoker;
+    private final ClientSaslContextImpl csc;
```

Comment:
I don't think that it's so well understood a concept that we should abbreviate the production code field name.